### DISCUSSION TITLE
Hololens 2

### DISCUSSION THREAD

**Discussion by Kiran53** [mlavik1/UnityVolumeRendering#106] [Q&A] [answered]
Does this work with hololens 2?

**Comment by mlavik1** [comment, marked as answer]
I don't own a Hololens, so I can't test, but there have been some discussions about it, such as: #71 
It seems like it should work if you change the stereo render mode settings to multi-pass.

I started investigating some issues with single pass instanced rendering (#101), but I no longer have a VR headset so I can't work on that either..
But again, from what I've heard it should probably work fine on Hololens 2 if you enable multi-pass rendering in Unity!

**Reply by Kiran53** [reply]
Thank you for the reply, will try it.

**Reply by mlavik1** [reply]
Great! Feel free to write here if it works or if you run into any other issues, since it can be useful for others to know :)

**Reply by Kiran53** [reply]
yes it works fine after changing render mode, thanks!

**Reply by mlavik1** [reply]
Great to hear! Thank you :)
